client/state/sites/reducer.js

```diff
@@ -269,6 +273,51 @@ export const items = withSchemaValidation( sitesSchema, ( state = null, action )
 	return state;
 } );
 
+/**
+ * Tracks all known site excerpt objects, indexed by site ID.
+ *
+ * @param  {object} state  Current state
+ * @param  {object} action Action payload
+ * @returns {object}        Updated state
+ */
+export const excerpts = withSchemaValidation( siteExcerptsSchema, ( state = null, action ) => {
+	if ( state === null && action.type !== SITE_EXCERPTS_RECEIVE ) {
```

Comment:
Won't that check be covered by the `switch` call below?

Noting that with `react-query` we can granularly set how much and for how long we want to cache things, and how fresh we want them to be, so that's another good reason to use it instead of Redux.